components/profile/shared/lists/GenericList.tsx

```diff
@@ -4,13 +4,14 @@ import React, { ReactElement } from "react";
 
 export type ListRow = {
   id: string;
+  isFirstRow: boolean;
```

Comment:
@EliseAndresen Burde kanskje hete default expanded? Jeg la til denne for agreements også, men litt rart når jeg sier at isFirstRow: false når den faktisk er first row :P

Reply:
Enig!